SIS_import/test/test_esb.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+require 'uri'
+require 'openssl'
+require 'net/http'
+require "json"
+require "base64"
+
+def httpClient(url)
+  url = URI(url)
+  http = Net::HTTP.new(url.host, url.port)
+  http.use_ssl = true
+  http.verify_mode = OpenSSL::SSL::VERIFY_PEER
+  return http
+end
+
+@esbToken=''
+# replace it with new esb key and secret
+@esbKey=''
+@esbSecret=''
+
+token_url= "https://apigw-tst.it.umich.edu/um/aa/oauth2/token"
```

Comment:
can this be read from configuration file?

Reply:
It could but this was added as part of testing. I have already implemented that feature for the curl script and ruby was just add on so I wanted to keep it simple